cmd/oras/push.go

```diff
@@ -276,22 +276,25 @@ func pushArtifact(dst *remote.Repository, pack packFunc, packOpts *oras.PackOpti
 	return root, nil
 }
 
-func isArtifactUnsupported(err error) bool {
+func noFallbackToOciImage(root ocispec.Descriptor, err error) bool {
```

Comment:
Making `isArtifactUnsupported` to `noFallbackToOciImage` might improve the readability of the invokers but make the readability of the implementation itself worse and easier to have bugs in it.

Reply:
Do you still have this concern after the refactoring?